spec/controllers/admin/ambassador_task_assignments_controller_spec.rb

```diff
@@ -0,0 +1,159 @@
+require "spec_helper"
+
+RSpec.describe Admin::AmbassadorTaskAssignmentsController, type: :request do
```

Comment:
If this is a request spec (which it appears to be?) I think it should be in the request specs folder

Reply:
How would you feel about consolidating both controller and request specs in `spec/controllers` (setting `type: :request` as needed) and refactoring the former to the latter incrementally? It'd be helpful to have both sets of tests in one place.

Reply:
Let's move the controller specs to "requests", rather than the other way around, since that is what I want to move toward.

This should be in a separate PR.

Reply:
Added an issue for this # #845